### PR TITLE
Add basic Postgres connection/query example

### PR DESCRIPTION
**Pull request by jason5122**
* Install JDBC dependency
* Add JDBC example code for connection and query

### CODE REVIEW

**Review by ellayunjikim — CHANGES_REQUESTED** [review]
Add a TODO for the databaseExperiment

File: src/main/java/dev/coms4156/project/teamproject/App.java [hunk 1]
```diff
@@ -34,6 +33,29 @@ public static void main(String[] args) {
      */
     @Override
     public void run(String[] args) {
+        databaseExperiment();
         System.out.println("Start up");
     }
+
+    private static void databaseExperiment() {
```
Comment (ellayunjikim) [inline src/main/java/dev/coms4156/project/teamproject/App.java:40]: Add a todo here

Comment (jason5122) [reply src/main/java/dev/coms4156/project/teamproject/App.java:42]: Done!

**Comment by jason5122** [comment]
This is obsolete since we're using JPA now.

**Pull request closed without merging**